.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Replace this text, including the symbols above and below it, with descriptive text about
+the change you are proposing. Please include a reference to any issues addressed or
+resolved in that text, for example "resolves #1".
+-->
+
+<!--
+IMPORTANT: As a contributor, we would like as much help as you can offer, but we only
+expect you to complete the steps in the "PR draft checklist" below. Maintainers are
+willing and ready to help pick it up from there!
+
+Please start by opening this Pull Request as a "draft". You can do this by
+clicking the arrow on the right side of the green "Create pull request" button. Follow
+the draft checklist below to move the PR out of draft state. If you accidentally created
+the PR as a non-draft, don't worry, you can still change it to a draft using the
+"Convert to draft" button on  the right side panel under the "Reviewers" section.
+-->
+
+<details><summary>Pull Request (PR) draft checklist - click to expand</summary>
+
+- [ ] Populate a descriptive title. Ensure the title does not look like "Updated
+      README.md".
+- [ ] Populate the body of the pull request with :
+    - A clear description of the change you are proposing.
+    - Links to any issues resolved by this PR with text in the PR description, for
+      example "closes #1".
+- [ ] Please review our
```

Comment:
Perhaps this should be the first item in the checklist?

Reply:
And the next one, "Ensure an issue exists representing the problem being solved in this PR.", could be the second item?

Reply:
f058822